scheduler/src/templates/stripe/success_image.html

```diff
@@ -21,10 +21,11 @@ <h1>Success!</h1>
         </div>
 
         <div class="description">
-            <h2>Using your hotspot</h2>
-            <p>Once the image has been downloaded, flash it onto a microSD card (we recommend <a href="https://www.raspberrypi.com/software/">Raspberry Pi Imager</a>, but any such tool will do).</p>
-            <p>Insert the card into your Raspberry Pi, plug it in, and wait for approximately 1-2 minutes during the initial boot-up process until you see the network status display <em>Ready</em> (or <em>Wikipedia</em> if you purchased Wikipedia image) appear in your wifi settings.</p>
-            <p>Connect to it and a captive portal should appear after a few seconds. If not, then open <a href="http://goto.kiwix.hotspot">http://goto.kiwix.hotspot</a> in your favourite browser.</p>
+            <h2>Downloading and your hotspot</h2>
+            <p>Bot links work, but torrent will restart where it stopped in case your connection breaks down and it will also check for file corruption after completion. <a href="https://www.transmissionbt.com">Transmission</a> is the client that we recommend.</p>
+            <p>Once the image has been downloaded, flash it onto a microSD card (we recommend <a href="https://www.raspberrypi.com/software/">Raspberry's Imager</a>, but any such tool will do).</p>
```

Comment:
Should we add a recommendation to let the check step complete?

Reply:
Seems like giving too much information. The check step is automatic, it is actually an effort to not perform it and I doubt people actively stop at this stage.

Reply:
I think otherwise. This step is very long (well correlated with size) and it's “just a verification” but if you think that's too much info let's leave it out. We have to ask them to redo it in the helpdesk anyway.